assembl/static2/js/app/components/administration/discussion/manageSectionsForm.jsx

```diff
@@ -0,0 +1,62 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { I18n } from 'react-redux-i18n';
+import { OverlayTrigger } from 'react-bootstrap';
+import SectionTitle from '../sectionTitle';
+import EditSectionForm from './editSectionForm';
+import { addSectionTooltip } from '../../common/tooltips';
+import * as actions from '../../../actions/adminActions/adminSections';
+
+type ManageSectionFormProps = {
+  sections: Object,
+  selectedLocale: string,
+  createSection: Function
+};
+
+const ManageSectionsForm = ({ sections, selectedLocale, createSection }: ManageSectionFormProps) => {
+  return (
+    <div className="admin-box">
+      <SectionTitle title={I18n.t('administration.sections.sectionsTitle')} annotation={I18n.t('administration.annotation')} />
+      <div className="admin-content">
+        <form>
+          {sections.map((id, index) => {
+            return <EditSectionForm key={id} id={id} index={index} locale={selectedLocale} nbSections={sections.size} />;
+          })}
+          <OverlayTrigger placement="top" overlay={addSectionTooltip}>
+            <div
+              onClick={() => {
+                return createSection(sections.size);
+              }}
+              className="plus margin-l"
+            >
+              +
+            </div>
+          </OverlayTrigger>
+        </form>
+      </div>
+    </div>
+  );
+};
+
+const mapStateToProps = (state) => {
+  const { sectionsInOrder, sectionsById } = state.admin.sections;
+  const sections = sectionsInOrder.filter((id) => {
```

Comment:
I think that this `.filter` is useless (if you removed the id from `sectionsInOrder` when a section is deleted which you should to be able to move sections properly)